R/endpoints.R

```diff
@@ -988,37 +1059,31 @@ fluview_meta <- function() {
   )
 }
 
-#' Obtains information on outpatient inluenza-like-illness (ILI) from Delphi's epidemiological data
-#'
-#' @details The full list of location inputs can be accsssed at <https://github.com/cmu-delphi/delphi-epidata/blob/main/src/acquisition/fluview/fluview_locations.py>.
-#'
-#' @examples
-#' call <- nowcast(location = "ca", epiweeks = epirange(201201, 202001))
-#' fetch_classic(call)
-#' @param locations a character string that indicates a location such as national, HHS region, census division, most states and territories, and etc.
-#' @param Takes in the form epirange(startdate,enddate), where startdate and enddate are of the form YYYYMMDD (can be passed as string or numeric).
-#' @return an instance of epidata_call
-#'
-#' @references API Documentation: <https://cmu-delphi.github.io/delphi-epidata/api/nowcast.html>.
-
 
-#' fetch fluview data
+#' Fetch fluview data
 #'
-#' Obtains information on outpatient inluenza-like-illness (ILI) from U.S. Outpatient Influenza-like Illness Surveillance Network (ILINet).
+#' Obtains information on outpatient inluenza-like-illness (ILI) from U.S. Outpatient Influenza-like Illness Surveillance
+#'   Network (ILINet).
 #'
-#' @details The full list of location inputs can be accsssed at <https://github.com/cmu-delphi/delphi-epidata/blob/main/src/acquisition/fluview/fluview_locations.py>.
+#' @details The full list of location inputs can be accsssed at
+#'   <https://github.com/cmu-delphi/delphi-epidata/blob/main/src/acquisition/fluview/fluview_locations.py>.
 #'
 #' @examples
 #' call <- fluview(regions = "nat", epiweeks = epirange(201201, 202001))
 #' fetch_classic(call)
-#' @param regions a character string that indicates a location such as national, HHS region, census division, most states and territories, and etc.
-#' @param epiweeks Takes in the form epirange(startdate,enddate), where startdate and enddate are of the form YYYYMMDD (can be passed as string or numeric).
-#' @param issues an optional parameter that takes the form YYYYMMDD specifying the exact issues to fetch.
-#' @param lag optionally specify the issue lag
-#' @param auth optional authentication
-#' @return an instance of epidata_call
-#'
-#' @references API Documentation: <https://cmu-delphi.github.io/delphi-epidata/api/fluview.html>.  For more information on ILINet, see <https://gis.cdc.gov/grasp/fluview/fluportaldashboard.html>.
+#' @param regions Character vector. The locations to fetch. Can we any string IDs in national, HHS region,
+#'   census division, most states and territories, and so on. Full list link below.
+#' @param epiweeks Epirange. The epiweeks to fetch in the form epirange(startweek,endweek), where startweek
+#'   and endweek are of the form YYYYWW (string or numeric).
+#' @param issues Epirange. Optionally, specify the issues to fetch in the form epirange(startweek,endweek), where startweek
```

Comment:
nitpick: Note whether something is optional in the same "sentence" as the type.

Reply:
I was just imitating how dplyr handled optional markings 🤷 , so I think I'm gonna keep as is for now